starlite/signature.py

```diff
@@ -322,6 +322,18 @@ def signature_parameters(self) -> Generator[SignatureParameter, None, None]:
                 continue
             yield SignatureParameter(self.fn_name, name, parameter)
 
+    def skip_parameter_validation(self, parameter: SignatureParameter) -> bool:
```

Comment:
Predicates should start in is, should etc. 

```suggestion
    def should_skip_parameter_validation(self, parameter: SignatureParameter) -> bool:
```